test/submission.browser.js

```diff
@@ -144,7 +143,7 @@ test('Ability to progress through the wizard is tied to validation', async t =>
 
   navigationHelper.navigateForward()
   // without this wait the tests sometimes fail on CI ¯\_(ツ)_/¯
-  navigationHelper.wait(1000)
+  navigationHelper.wait(2000)
```

Comment:
Is there an issue to investigate this? Doesn't seem very scalable if we've had to increase this and would be better to fully understand why its required.

Reply:
I agree what this is waiting for? It should be a condition within a polling loop that reaches a steady state, such as the presence of a new element in the DOM or a class being added to an existing element.

Reply:
Yeah agree lets add this to #1931  ... I'm loathed to change too much more with this PR.

Reply:
Yes, this is getting out of hand :+1: